### PR TITLE
Simplify labels

### PR DESCRIPTION
**Pull request by luchaos**
A more unified, visually consistent inverted version of labels that does not have too many variants and uses existing color variables.

I know you wanted to make a distinction between types, @EmoonX, but i feel it's still to early to introduce such changes before the tag system is in place. Tags will have to then emulate every behaviour that's added until then. Hence why i wanted those colors to be gone previously; to keep it simple for now and make the transition easier.

Screenshot 2023-01-07 at 02 57 49
Screenshot 2023-01-07 at 02 58 07
Screenshot 2023-01-07 at 02 57 58
Screenshot 2023-01-07 at 03 05 05
Screenshot 2023-01-07 at 03 05 20

### CODE REVIEW

**Comment by luchaos** [comment]
Added another commit to put labels at the end of game titles.

As discussed before @Jamiras this messes with the perceived ordering on game lists, however makes it much easier to parse. What do you think, should we keep it nontheless?

Screenshot 2023-01-07 at 03 15 42
Screenshot 2023-01-07 at 03 15 51
Screenshot 2023-01-07 at 03 16 06

**Review by Jamiras — APPROVED** [review]
> this messes with the perceived ordering on game lists, however makes it much easier to parse. What do you think, should we keep it nontheless?

It's pretty clear there's a separation between non-tagged and tagged items, but the subtle changes from Hack to Homebrew to Prototype are less clear. I agree the decorator looks better at the end, and it's really only the sorting on the game list that's affected, so I think it's okay to make the change.

File: resources/css/achievement-set.css [hunk 1]
```diff
@@ -3,12 +3,13 @@
     margin-left: .1em;
 }
 
+/*
```
[the remark this answers is not in the document]
Comment (luchaos) [reply resources/css/achievement-set.css:6]: There was a stray comment opener below that did nothing. Removed that to have a matching end 👍

**Comment by EmoonX** [comment]
A suggestion: since we're going for "unified" and "visually consistent" anyway, how about using the same adjacent text color for tags? This way the blend together and flow like a single element (for added minimalism).





The special CSS value `currentColor` works well for this. Even has the added bonus of highlighting it all on link hover:

**Comment by EmoonX** [comment]
Regarding subsets: as expected, their labels look the same as categories' now. Even worse, they're all positioned on the right, which further blurs the difference between the two. Basically, they can only be told apart by the `Subset - ` string at the beginning.

I know we're doing one-style-fits-all, but I thought of something nice that can be applied to subset tags while still keeping in trend with the philosophy and chosen color scheme (this is just a mock-up for now):

image
image

Gets rid of the annoying dash as a bonus. Further on it can even be expanded to fit other tags that match the `Something - lorem ipsum` pattern.

If you feel like these are good changes, I can open a PR to address them.

**Comment by luchaos** [comment]
> how about using the same adjacent text color for tags? This way the blend together and flow like a single element (for added minimalism).

I agree; for some reason I thought this was already the case. I gave it a try and had to shuffle around a few things to make `currentColor` work. I'll state a PR for it as it includes some further simplification.


> image image

I really like that idea - I'd love to see a PR for this. I'm not quite sure how this will work with currentColor, probably an overlay colored with a transparent highlight color?

**Comment by EmoonX** [comment]
I was playing a bit with it up until now and got both ideas to work here. If it's okay with you, I can make a single PR addressing both suggestions.

>probably an overlay colored with a transparent highlight color?

Yep, that's basically what I ended up doing 😋

**Pull request merged**